financials/reports/funding_transparency_2020Q3.md

```diff
@@ -0,0 +1,71 @@
+# 2020Q3 Funding Transparency Report
+
+Last updated: Oct 26 2020
+
+## Introduction
+This document provides a high level overview of Grin's financials for Q3 of 2020 (Jul 01 - Sep 30). The purpose is to keep the community up to date on funding activity in transparent and digestible way. The data is derived from what is reported in the financial logs under [/grin-pm/financials](https://github.com/mimblewimble/grin-pm/tree/master/financials).
+
+### Notes
+
+1. Data was up to date at the time of writing, but it's still possible that there have been changes and updates in the logs since.
+2. USD conversion rates are derived from the closing price of the asset on the day in question according to Coinmarketcap.com unless otherwise stated. 
+
+## Status before the period
+
+### Disposable funds
+
+At the end of Jun 30 2020, Grin held disposable funds to the equivalent of $898,220.37 in the following assets:
+
+Currency | Amount | USD x-rate Jun 30 2020 | USD Equivalent | Wallet address(es)
+|---|---:|---:|---:|---|
+GRIN | 3770.71555489 | $0.415295 | $1,565.96 | https://donations.grin-tech.org
+BTC | 98.12381179 | $9,137.99 | $896,654.41 | 9hgKkhfPFpwKRQ6yARGwU2uuVay69NKu4<br />bc1qecw954uappnwhle6tmafqskn6vxmpr6lmmuwfwldphfxplcshgnssy9kmx
+| | | | $898,220.37 |
+
+### Additional details
+* See the [2020Q2 report](funding_transparency_2020Q2.md).
+
+## Activity during the period
+
+### Received
+
+During Jul 01 - Sep 30 2020, Grin received the following assets: 
+
+Currency | Amount | Wallet address(es)
+|---|---:|---|
+GRIN | 0 | https://council-donations.yeastplume.org
+BTC | 10.0 | bc1qecw954uappnwhle6tmafqskn6vxmpr6lmmuwfwldphfxplcshgnssy9kmx
+
+### Spent
+
+During Jul 01 - Sep 30 2020, Grin spent the following assets:
+
+Currency | Amount | 
+|---|---:|
+BTC | 3.58 | 
+
+### Comments
+* Grin received a single donation in this period, which was the 10 BTC donation by Nervos Foundation.
+* Aside from 0.03 BTC for forum costs, all spend was related to funding requests, specifically:
+   * [@paouky](../../notes/20200728-meeting-governance.md#decision-approve-paouky-funding-request)
+   * [@i1skn](../../notes/20200908-meeting-governance.md#decision-approve-i1skn-funding-request)
+   * [@davidtavarez](../../notes/20200908-meeting-governance.md#decision-approve-dtavarez-funding-request)
+* As per previous report, @yeastplume, @antiochp, and @lehnberg were already funded in Q2 for this quarter.
+
+## Status after the period
+
+### Disposable funds
+
+At the end of Sep 30 2020, Grin held disposable funds to the equivalent of $898,220.37 in the following assets:
+
+Currency | Amount | USD x-rate Sep 30 2020 | USD Equivalent | Wallet address(es)
+|---|---:|---:|---:|---|
+GRIN | 3770.71555489 | $0.306694 | $1,156.46 | https://donations.grin-tech.org
+BTC | 104.54381179 | $10,784.49 | $1,127,451.69 | 9hgKkhfPFpwKRQ6yARGwU2uuVay69NKu4<br />bc1qecw954uappnwhle6tmafqskn6vxmpr6lmmuwfwldphfxplcshgnssy9kmx
```

Comment:
I think the total was 104.514006 BTC here.

Reply:
Confirmed with BlockCypher here https://api.blockcypher.com/v1/btc/main/addrs/bc1qecw954uappnwhle6tmafqskn6vxmpr6lmmuwfwldphfxplcshgnssy9kmx see `ref_balance`